src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java

```diff
@@ -95,34 +95,41 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example of using escapes with trail comment:
```

Comment:
please fix

```
 * String unitAbbrev = "μs";      // Best: perfectly clear even without a comment.
 * String unitAbbrev = "&#92;u03bcs"; // Poor: the reader has no idea what this is.
```

to be (combine two example blocks)
```
 * String unitAbbrev = "μs";      // OK, perfectly clear even without a comment.
 * String unitAbbrev = "&#92;u03bcs"; // violation, the reader has no idea what this is.
 * return '\ufeff' + content; // OK, an example of non-printable, control characters (byte order mark). 
```

please use the same code block for each config. Please attentively 
https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-examples "ATTENTION"

you will need to add to common code block some specific case like
`String unitAbbrev = "\u03bc\u03bcs"; // Violation, not all characters are escaped ('s').`

Reply:
done